Style guide enhancements (#1410)
OWASP/owasp-mstg: style_guide.md

### American spelling and terminology

Use American spelling and terminology.
Change all British spelling and terminology to the American equivalents where applicable. This includes "toward" (US) vs. "towards" (UK), "among" (US) vs. "amongst" (UK), "analyze" (US) vs. "analyse" (UK), "behavior" (US) vs "behaviour" (UK), etc.

### Plurals

Adhere to standard grammar and punctuation rules when it comes to pluralization of typical words.

The plural of calendar years does not take the apostrophe before the “s”. For example, the plural form of 1990 is 1990s.

### Title Capitalization

We follow the title case rules from the "Chicago Manual of Style":  

- Capitalize the first and last word in a title, regardless of part of speech
- Capitalize all nouns (app, encryption, package), pronouns (you, she, it), verbs (analyze, compile, inspect), adjectives (active, insecure, weak), adverbs (immediately, quietly), and subordinating conjunctions (as, because, although)
- Lowercase “to” as part of an infinitive
- Lowercase all articles (a, the), prepositions (to, at, in, with), and coordinating conjunctions (and, but, or)

> When in doubt, you can verify proper capitalization on [https://titlecaseconverter.com/](https://titlecaseconverter.com/).

Reviewer: Add alt-text
```suggestion
When in doubt, you can verify proper capitalization on [https://titlecaseconverter.com/](https://titlecaseconverter.com/ "Title Case Converter").
```

### Abbreviations

Abbreviations include acronyms, initialisms, shortened words, and contractions.

- Spell out the term the first time it's used, followed by the abbreviation in parentheses. Example: Mobile Security Testing Guide (MSTG). Subsequent usages in the same chapter may include the abbreviation only.
- If it only appears once in the content, spell out the term instead of using the abbreviation.
- In titles and headings, use the abbreviation but be sure to properly introduce it (see above) in the text that follows.

> - Use "a" or "an" depending on the pronunciation (as with regular words). Example: a DLL, an APK, a URL.

Reviewer: ```suggestion
- Use "a" or "an" depending on the pronunciation of the acronym. Example: a DLL, an APK, a URL, a SQL.
```

I don't remember which one, but there are a few that have a different pronunciation depending on if you say the abbreviation or the full words. Let's make the acronym the default one.

I also added the SQL example as it should be pronounced "sequel", but many people pronounce it as es-que-el.
